examples/gnrc_border_router/Makefile.board.dep

```diff
@@ -0,0 +1,6 @@
+# Put here board specific dependencies
```

Comment:
```suggestion
# Put board specific dependencies here
```

is grammatically more correct btw.

Reply:
Fixed here and in other Makefile.board.dep files.